documentation/assemblies/assembly-managing-users-oc.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+//master.adoc
+:parent-context: {context}
+:context: managing-users-oc
+:cmdcli: oc
+
+[id='managing-users-{context}']
+= Managing users on OpenShift
+
+{ProductName} user management is only supported when using the `standard` authentication service. On OpenShift, users can
+be managed using the OpenShift command-line tools.
+
+.Prerequisites
+
+* {ProductName} is installed with custom resources enabled
```

Comment:
I think we can remove this prerequisite, as it is enabled by default and unlikely to not be enabled.

Reply:
Done